stagecraft/apps/datasets/tests/models/test_data_set.py

```diff
@@ -22,6 +23,8 @@
 from stagecraft.apps.datasets.models.data_set import ImmutableFieldError, \
     DataSetQuerySet
 
+from stagecraft.apps.users.models import User
+
```

Comment:
Can we remove the blank line? And perhaps the blank line between this import and the previous ones?